src/System.Security.Cryptography.Algorithms/ref/System.Security.Cryptography.Algorithms.cs

```diff
@@ -601,8 +601,6 @@ public abstract partial class TripleDES : System.Security.Cryptography.Symmetric
     {
         protected TripleDES() { }
         public override byte[] Key { get { throw null; } set { } }
-        public override System.Security.Cryptography.KeySizes[] LegalBlockSizes { get { throw null; } }
```

Comment:
It is up for debate see https://github.com/dotnet/corefx/pull/18138 but I'm still in favor of allowing them to be removed.